flink-core/src/test/java/org/apache/flink/util/UserClassLoaderJarTestUtils.java

```diff
@@ -35,7 +33,9 @@
 import java.util.jar.JarOutputStream;
 
 /** Mainly used for testing classloading. */
-public class TestUserClassLoaderJar {
+public class UserClassLoaderJarTestUtils {
+
+    private UserClassLoaderJarTestUtils() {}
```

Comment:
Better to throw an exception when the constructor is invoked, one reference can be flink-core/src/main/java/org/apache/flink/util/ClassLoaderUtil.java